internal/dataplane/configfetcher/kongrawstate_test.go

```diff
@@ -344,6 +344,8 @@ func ensureAllKongStateFieldsAreTested(t *testing.T, testedFields []string) {
 		"Plugins",
 		// Licenses are injected from the license getter rather than extracted from the last state.
 		"Licenses",
+		// CustomEntities are not supported yet because go-database-reconciler does not include custom entities.
```

Comment:
Do we want to create an issue for this and reference it here?

Reply:
OK, will do,

Reply:
Already there: #6054